bnm/recon/io/factory.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+
+import os
+from bnm.recon.io.annotation import H5AnnotationIO, AnnotationIO
+from bnm.recon.io.surface import GiftiSurfaceIO, FreesurferIO, H5SurfaceIO
+from bnm.recon.io.volume import VolumeIO, H5VolumeIO
+from bnm.recon.model.constants import GIFTI_EXTENSION, H5_EXTENSION
+
+
+class IOFactory(object):
```

Comment:
As this is more than a Factory, I recommend either renaming this class or splitting into two parts:
- IOFactory
- IO(main)